shakenfist/images.py

```diff
@@ -68,7 +76,7 @@ def new(cls, url, checksum=None):
             'version': cls.current_version
         })
         i = Image.from_db(uuid)
-        i._db_set_attribute('state', {'state': 'initial'})
+        i._db_set_attribute('state', {'state': None})
         i.update_state('initial')
```

Comment:
Why do this in two db writes? Do you see an advantage I'm missing?

Reply:
Minimal touching of the code. Just keeping this change to what was required. The change to that line was only the initial value.

I do agree that it is unnecessary. The related functions look like they will handle the missing DB entry.

Reply:
Ok, I will merge this and then revisit.